specs/docker/remove_docker_images_from_local.yaml

```diff
@@ -0,0 +1,14 @@
+---
+name: Remove docker images from local system
+command: docker rmi {{image_ids}}
+tags:
+  - docker
+description: Remove docker images currently stored on your system
+arguments: 
+  - name: image_id
+      description: "This image ID can be found by using the `docker images ls` command"
+      default_value: ~
```

Comment:
@guel-codes I think we should remove the default argument for this workflow; it'll be safer if the user has to explicitly specify the image to remove.

Reply:
@elviskahoro this has been updated sorry for the delay